### PR TITLE
Task: add remaining fields (maximum queued executions, on child error)

### LINKED ISSUE #90: Define maximum queued executions for Task
It would be useful to be able to define the number of max queued executions in the `valohai.yaml`.

### PR DESCRIPTION
**Pull request by akx**
Fixes #90

### CODE REVIEW

File: examples/task-example.yaml [hunk 1]
```diff
@@ -49,6 +49,8 @@
     type: RANDOM-SEARCH # will be normalized
     execution-batch-size: 42
     execution-count: 420
+    maximum-queued-executions: 17
+    on-child-error: stop_all-AND_error
```
Comment (axelauvinen) [inline examples/task-example.yaml:53]: Opinion: Is this written intentionally wrong to represent fail-safe working in `cast` method? Ok, but maybe bit weird for an example file 🤔 I would consider adding weirdness into separate test file and merging those into better written example

**Review by axelauvinen — APPROVED** [review]
LGTM, opinionated example botherment

File: examples/task-example.yaml (hunk 1 above)
Comment (akx) [reply examples/task-example.yaml:53]: Maybe I'll add a comment there too, sure :)

**Review by axelauvinen — APPROVED** [review]
🚀

**Pull request merged**